### ISSUE TITLE
Couldn't up the instance everytime when we are running frequently with out errors

### ISSUE THREAD

**Issue by saneysrikanth** [OpenCTI-Platform/opencti#458]
## Description

couldn't up the instance everytime when we are running frequently.

## Environment

1. OS (where OpenCTI server runs): { Ubuntu 18.4. }
2. OpenCTI version: { e.g. OpenCTI 2.1.4}
3. OpenCTI client: { e.g. python(PYCTI }

## Expected Output

Make use of Opencti instance without errors and use PYCTI.

## Actual Output

Unable to start instance,though it starts facing some grakn issues and PYCTI issues

![image](https://user-images.githubusercontent.com/45160537/73519462-2eee3d80-4427-11ea-958e-61d0e9948ee3.png)

 
## Additional information
I tried the following

- I tried stopping and remove opencti docker containers,but no use of it.
- I also tried docker-compose down -v (removing volumes and restarting) but all the information is resetted.

**Comment by graememeyer** [comment]
This looks like the same issue as #322. There's an issue with the documentation / default config for the port number of OpenCTI.

The fix is to change the port number OpenCTI is using in your docker-compose.yml file.

You can use this SED command to do this: `sed -i 's/8080/4000/g' docker-compose.yml`

You also need to make sure your tokens and UUIDs (in the same config file) are correct. You can see an example of correctness in the docker-compose_test.yml file (a pre-configured test example). I'm not sure if it's required, but when I use the test file (by replacing the docker-compose.yml with the the test config), I also change two remaining "ChangeMe*" values with these two SED commands: 

```
sed -i 's/ChangeMeAccess/NewAccessINSERTRANDOMSTRINGHERE/g' docker-compose.yml
sed -i 's/ChangeMeKey/NewAccessKeyINSERTRANDOMSTRINGHERE/g' docker-compose.yml
```

tl;dr if you have same issue as #322, run the three listed SED commands in your docker directory


Edit: You do also have to leave it for quite a while before the errors stop. Docker seems to start all the compartments in totally the wrong order and has to iterate through the entire set a few times before it starts working. I'd say leave it at least 20 minutes. 

You should eventually see the errors turn into info and other log messages indicating that the content is being constructed. In my experience this takes many hours, even on powerful systems. You can access the web-gui (now at the new port number you inserted) while the content is being built though. Essentially, once the errors stop.

**Comment by SamuelHassine** [comment]
Hello, please try with the latest version and the documentation. If you still have an error please reopen this issue.

**Comment by stephent23** [comment]
Hello, I seem to be having this same issue. I have tried the steps above and followed the documentation but still have the same issue. 

As per the screenshot below the five containers at the top just continue to restart. When I have run this with docker-compose --compatibility up, the screen output displays that the OpenCTI API has failed to start. 
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/8761562/77915524-8ece5c80-728f-11ea-942f-16a7ab870e2f.png">

Any help appreciated.
